src/core/session.cpp

```diff
@@ -73,15 +73,15 @@ class standard_logger_impl : public logger_impl
 } // namespace anonymous
 
 session::session()
-    : once(this), prepare(this), query_transformation_(NULL),
+    : once(this), prepare(this),
       logger_(new standard_logger_impl),
       uppercaseColumnNames_(false), backEnd_(NULL),
```

Comment:
While transitioning to a modern cpp standard, it would probably also be good to replace all `NULL` with `nullptr`

Reply:
Yes, I'm going to use clang-tidy to do this later, but let's merge this one first.